ui/k-info-panel/main.kytos

```diff
@@ -0,0 +1,307 @@
+<template>
+    <div v-if="render_value > 0">
+        <div class="maintenance_container">
+            <div class="maintenance-table no-compact">
+                <div class="maintenance-table-divisor">
+                    <table id="maintenance-table-list-windows">
+                        <thead>
+                            <tr>
+                                <template v-for="(header, index) in table_headers">
+                                    <th scope="col" class="header" @click="sort(index)">
+                                        {{ header }}
+                                        <span class="order-dir" v-if="current_sort === index">
+                                            {{ spanDir(index) }}
+                                        </span>
+                                    </th>
+                                </template>
+                            </tr>
+                            <tr class="header-filter">
+                                <template v-for="header in table_headers_id">
+                                    <th scope="col">
+                                        <input :id="header + '_search'" @keyup="filterWindowList"></input>
+                                    </th>
+                                </template>
+                            </tr>
+                        </thead>
+                        <tbody>
+                            <tr v-for="row in rowsOfPage">
+                                <template v-for="column in row">
+                                    <td>{{column}}</td>
+                                </template>
+                            </tr>
+                        </tbody>
+                    </table>
+                </div>
+            </div>
+        </div>
+    </div>
+    <div v-else>
+        <div class="maintenance_container">
+            <p class='empty-window-list'>No maintenance windows available.<p>
```

Comment:
@Aduan002 could you also implement an extra conditional `<p>` when the component it's loading and the request hasn't finished yet? The backend is supposed to always reply in a consistent and reliable manner, but delayed responses might happen. Check out the [`flag_loading`](https://github.com/kytos-ng/mef_eline/blob/master/ui/k-info-panel/list_connections.kytos#L56) that we've got on `mef_eline` list_connections component, something similar to this.